server.js

```diff
@@ -22,4 +22,19 @@ app.get('/api/v1/residents', async (request, response) => {
   } catch(error) {
     response.status(500).json({ error });
   }
-});
\ No newline at end of file
+});
+
+app.get('/api/v1/residents/:semester', async (request, response) => {
```

Comment:
Nice, this will be helpful in querying by semester.
Let's not forget to: 
* Pull back down
* Push up to heroku
* Postentially migrage, but definitely seed